siatest/renter.go

```diff
@@ -58,6 +58,15 @@ func (tn *TestNode) DownloadByStream(rf *RemoteFile) (data []byte, err error) {
 	return
 }
 
+// Stream uses the streaming endpoint to download a file.
+func (tn *TestNode) Stream(rf *RemoteFile) (data []byte, err error) {
+	data, err = tn.RenterStreamGet(rf.siaPath)
+	if err == nil && rf.checksum != crypto.HashAll(data) {
+		err = errors.New("downloaded bytes don't match requested data")
+	}
+	return
+}
```

Comment:
Stream function ideally jumps around a little bit as well, to mimic a user seeking. You'll need a checksum of each part of the file of course to do that.